overlay/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ option(BUILD_OVERLAY_XCOMPILE "Build an x86 overlay" OFF)
 if(BUILD_OVERLAY_XCOMPILE)
 	cmake_minimum_required(VERSION 3.15)
 
-	set(version "1.4.0" CACHE STRING "Project version")
+	set(version "1.5" CACHE STRING "Project version")
```

Comment:
This shouldn't be set explicitly. Instead we should just use the cmake variable holding the main project's version number